src/sections/aliases.tsx

```diff
@@ -67,11 +67,11 @@ function AliasesPage() {
       <button
         disabled={aliases !== null && Object.keys(aliases).length === 0}
         onClick={() => {
-          if (aliases) {
+          if (aliases.data) {
```

Comment:
there are a couple places where .data is being used, i think you can decide if which ones should be changed.